terraform/connection_config.go

```diff
@@ -6,12 +6,12 @@ import (
 )
 
 type terraformConfig struct {
-	Paths         []string `cty:"paths" steampipe:"watch"`
-	PlanFilePaths []string `cty:"plan_file_paths" steampipe:"watch"`
+	ConfigurationFilePaths []string `cty:"configuration_file_paths" steampipe:"watch"`
```

Comment:
@Subhajit97 I think we should not remove the `paths` argument entirely, rather we need to deprecate it and avoid breaking changes right away.